src/Backups/DDLAdjustingForBackupVisitor.h

```diff
@@ -1,6 +1,6 @@
 #pragma once
 
-#include <Interpreters/InDepthNodeVisitor.h>
+#include <Parsers/InDepthNodeVisitor.h>
```

Comment:
Not clear why we moved `InDepthNodeVisitor` to `Parsers`, I dont see any new usages for this, maybe I missed something ?

Reply:
It's a part of an older solution when the masking was implemented on base of InDepthNodeVisitor. I think I'll move it back to Interpreters

Reply:
done